main/monmon.rb

```diff
@@ -26,17 +51,8 @@
 
```

Comment:
the process should be this (pseudocode):

```
table = read_csv
result = process(table)
print(result)
```

"table" - is array of hashes, this way the "process" function won't depend on CSV what makes it "clean" and easy to test, because in tests you can created this array and hashes manually without csv file
the "process" function should go through all rows and do all calculations. also(ideally it should be in a separate file so that when you run tests they don't run the whole program and other things that are not needed for this function testing). start writing tests for this function
"print" function should just take results from process and do printing into stdout. in current code it's just all places that call `puts` - it's necessary create "print" function right now, you can do this later in a separate PR